[2022-03-04 16:00:41,759] INFO Expiring session 0x1000ab282fb0000, timeout of 18000ms exceeded (org.apache.zookeeper.server.ZooKeeperServer)
[2022-03-04 16:00:41,057] WARN Client session timed out, have not heard from server in 6763885ms for sessionid 0x1000ab282fb0000 (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:45,069] INFO Client session timed out, have not heard from server in 6763885ms for sessionid 0x1000ab282fb0000, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,391] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,437] INFO Socket connection established, initiating session, client: /127.0.0.1:59735, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,470] INFO Invalid session 0x1000ab282fb0000 for client /127.0.0.1:59735, probably expired (org.apache.zookeeper.server.ZooKeeperServer)
[2022-03-04 16:00:46,528] WARN Unable to reconnect to ZooKeeper service, session 0x1000ab282fb0000 has expired (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,542] INFO Unable to reconnect to ZooKeeper service, session 0x1000ab282fb0000 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,529] INFO EventThread shut down for session: 0x1000ab282fb0000 (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,553] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2022-03-04 16:00:46,556] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2022-03-04 16:00:46,557] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@f6efaab (org.apache.zookeeper.ZooKeeper)
[2022-03-04 16:00:46,579] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2022-03-04 16:00:46,590] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,819] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,840] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2022-03-04 16:00:46,845] INFO Socket connection established, initiating session, client: /0:0:0:0:0:0:0:1:60056, server: localhost/0:0:0:0:0:0:0:1:2181 (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,850] INFO Session establishment complete on server localhost/0:0:0:0:0:0:0:1:2181, sessionid = 0x1000ab282fb0001, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:00:46,854] INFO Updated cache from existing FinalizedFeaturesAndEpoch(features=Features{}, epoch=0) to latest FinalizedFeaturesAndEpoch(features=Features{}, epoch=0). (kafka.server.FinalizedFeatureCache)
[2022-03-04 16:00:46,855] INFO Stat of the created znode at /brokers/ids/0 is: 294,294,1646389846852,1646389846852,1,0,0,72069355855872001,214,0,294
 (kafka.zk.KafkaZkClient)
[2022-03-04 16:00:46,856] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://DESKTOP-E4DA52B:9092, czxid (broker epoch): 294 (kafka.zk.KafkaZkClient)
[2022-03-04 16:01:32,746] WARN Session 0x1000ab282fb0001 for server localhost/0:0:0:0:0:0:0:1:2181, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:77)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:365)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1223)
[2022-03-04 16:01:34,003] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:36,055] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused: no further information (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:37,740] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:39,787] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused: no further information (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:41,101] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:41,879] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2022-03-04 16:01:41,881] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2022-03-04 16:01:43,130] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused: no further information (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:43,240] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2022-03-04 16:01:45,250] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:47,309] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused: no further information (org.apache.zookeeper.ClientCnxn)
[2022-03-04 16:01:49,013] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
